src/symfony/provider/ConsoleContainerProvider.ts

```diff
@@ -27,7 +27,7 @@ export class ConsoleContainerProvider implements ContainerProviderInterface {
     }
 
     provideServiceDefinitions(): Promise<ServiceDefinition[]> {
-        return this._executeCommand<ServiceDefinition>(["debug:container", "--show-private"], (obj) => {
+        return this._executeCommand<ServiceDefinition>(["debug:container", "--show-hiden"], (obj) => {
```

Comment:
@thorne51 - you misspelled "hidden" in your fix.

Reply:
Thanks @indiegenus , had it right and somehow messed it up :smile: , fixed in ba7d611eb62dec483b1992be3fe1a0fc511274e5